test/e2e/setup.go

```diff
@@ -153,6 +313,31 @@ func newClientSet(ctx context.Context) (*clientSet, error) {
 	}, nil
 }
 
+func setupSelenium(ctx context.Context) error {
+	log.Infof("Starting Selenium Grid")
+	cmd := exec.CommandContext(ctx, "docker", "pull", "selenium/standalone-edge:latest")
+
+	output, err := cmd.CombinedOutput()
+
+	log.Infof("Selenium Image Pull Output : %s\n", output)
+
+	if err != nil {
+		log.Fatalf("Error occurred pulling selenium image\n Output: %s\n Error: %s\n", output, err)
+	}
```

Comment:
Let's move this around
```suggestion
	output, err := cmd.CombinedOutput()
	if err != nil {
		log.Fatalf("Error occurred pulling selenium image\n Output: %s\n Error: %s\n", output, err)
	}

	log.Infof("Selenium Image Pull Output : %s\n", output)

Reply:
Sufficiently shuffled